salt/utils/timed_subprocess.py

```diff
@@ -59,24 +57,24 @@ def __init__(self, args, **kwargs):
                     try:
                         args = shlex.split(args)
                     except AttributeError:
-                        args = shlex.split(six.text_type(args))
+                        args = shlex.split(str(args))
                 str_args = []
                 for arg in args:
-                    if not isinstance(arg, six.string_types):
-                        str_args.append(six.text_type(arg))
+                    if not isinstance(arg, str):
+                        str_args.append(str(arg))
                     else:
                         str_args.append(arg)
                 args = str_args
             else:
-                if not isinstance(args, (list, tuple, six.string_types)):
+                if not isinstance(args, (list, tuple, (str,))):
```

Comment:
```suggestion
                if not isinstance(args, (list, tuple, str)):
```
Yes, I know, our automated code upgrade did not do a great job...

Reply:
Good catch, tnank you